deploy/deployment.yaml

```diff
@@ -17,8 +17,8 @@ spec:
       serviceAccountName: aws-account-shredder
       containers:
         - name: aws-account-shredder
-          image: quay.io/agautam/aws-account-shredder:latest
-          imagePullPolicy: always
+          image: quay.io/app-sre/aws-account-shredder
```

Comment:
why was latest removed?

Reply:
I just copied what was on the [pipeline](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/osd-operators/cicd/saas/saas-aws-account-shredder.yaml#L30)

Reply:
The pipline just replaces the image name in our template. Our template uses latest as the tag. Please add latest back here